src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,374 @@
+
+  position: relative;
+  display: flex;
+  justify-content: center;
+  background-color: var(--pf-c-sequence--BackgroundColor);
+
+  // Hide scrollbars
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.pf-c-sequence__control {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  padding:
+    var(--pf-c-sequence__control--PaddingTop)
+    var(--pf-c-sequence__control--PaddingRight)
+    var(--pf-c-sequence__control--PaddingBottom)
+    var(--pf-c-sequence__control--PaddingLeft);
+  border-top: var(--pf-c-sequence__control--BorderTopHeight) solid var(--pf-c-sequence__control--BorderTopColor);
+  border-bottom: var(--pf-c-sequence__control--BorderBottomHeight) solid var(--pf-c-sequence__control--BorderBottomColor);
+
+  &.pf-m-hover,
+  &:hover {
+    border-top: var(--pf-c-sequence__control--hover--BorderTopHeight) solid var(--pf-c-sequence__control--hover--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--hover--BorderBottomHeight) solid var(--pf-c-sequence__control--hover--BorderBottomColor);
+  }
+
+  &.pf-m-active,
+  &:active {
+    border-top: var(--pf-c-sequence__control--active--BorderTopHeight) solid var(--pf-c-sequence__control--active--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--active--BorderBottomHeight) solid var(--pf-c-sequence__control--active--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopHeight) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomHeight) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+}
+
+.pf-c-sequence__control-left {
+  left: var(--pf-c-sequence__control-left--PositionLeft);
+  background-color: var(--pf-c-sequence__control-left--BackgroundColor);
+  border-right: var(--pf-c-sequence__control-left--BorderRightWidth) solid var(--pf-c-sequence__control-left--BorderRightColor);
+}
+
+.pf-c-sequence__control-right {
+  right: var(--pf-c-sequence__control-right--PositionRight);
+  background-color: var(--pf-c-sequence__control-right--BackgroundColor);
+  border-left: var(--pf-c-sequence__control-right--BorderLeftWidth) solid var(--pf-c-sequence__control-right--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  color: var(--pf-c-sequence__control-icon--Color);
+}
+
+// Horizontal list
+.pf-c-sequence__list {
+  display: inline-flex;
+  overflow: scroll;
+}
+
+// Change default styles for the list and list items
+.pf-c-sequence__list .pf-c-sequence__item {
+  list-style-type: none;
+
+  // added so that the list will be read by screen reader
+  &::before {
+    width: 1px;
+    content: "";
+  }
+}
+
+// Items inside of list
+.pf-c-sequence__item {
+  max-width: var(--pf-c-sequence__item--MaxWidth);
+  margin-right: var(--pf-c-sequence__item--MarginRight);
+  margin-left: var(--pf-c-sequence__item--MarginLeft);
```

Comment:
just curious why you use left/right margin, and not something like `&:not(:last-child) { margin-right: var(--pf-c-sequence__item--MarginRight); } ` and change the var to 2px.

Reply:
added &:not(:last-child) for margin-right and added &:not(:first-child) for margin-left, to ensure there are no extra margins. IMO changing the var to 2px creates too large of a space in between them as compared to the docs. What was your thought behind changing it to 2px?

Reply:
oh sorry... what I mean is you're adding a margin-left and margin-right on all of the elements, which creates a 2px gap between them. why not just use a 2px margin-right on all but the last element, or 2px margin-left on all but the first?

Reply:
ohh thanks for clarifying! making those changes